core/src/main/scala/io/unsecurity/AbstractContentTypeMatcher.scala

```diff
@@ -41,4 +42,38 @@ abstract class AbstractContentTypeMatcher[F[_]: Monad] extends AbstractMethodMat
     }
   }
 
+  def matchAcceptContentType[A](responseAlternatives: NonEmptyList[ResponseAlternativeForContent[A]]): Directive[F, A] = {
+    NonEmptyList
+      .fromList(
+        responseAlternatives.collect{ case ResponseAlternativeForContent(Some(responseContentType), a ) => responseContentType.mediaType -> a }
+      )
+      .map[Directive[F, A]]{ contentTypeResponses =>
+        Directive.request[F].flatMap{ req =>
```

Comment:
Dette er helt unødvendig. Directive er en funksjon fra `Request[F] => Result[F, A]`
Se gjerne på RequestOps for noen hjelpere her